test/integration-ebpf/src/bpf_probe_read.rs

```diff
@@ -1,5 +1,5 @@
-#![no_std]
-#![no_main]
+#![cfg_attr(target_arch = "bpf", no_std)]
```

Comment:
can we write a macro that expands to all of this? these two, the panic handler, and the main?

Reply:
Not for the global inner attributes unfortunately:

![image](https://github.com/user-attachments/assets/56533918-d0e3-40ad-86c3-121f9aa347af)

See https://github.com/rust-lang/rust/issues/54726. We can encapsulate the panic handler and the main-stub.